spec/mongoid/query_cache_spec.rb

```diff
@@ -2,6 +2,7 @@
 # encoding: utf-8
 
 require "spec_helper"
+require "support/session_registry" 
```

Comment:
This shouldn't be needed if spec helper is required?

Reply:
Yup, good catch!